package.json

```diff
@@ -10,6 +10,7 @@
     "test": "jest --coverage --verbose",
     "posttest": "cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js",
     "postversion": "npm publish",
+    "start": "webpack-dev-server",
```

Comment:
This is the same as the `dev` command on line 7 🙂 So you can just type `npm run dev`